force-app/main/default/classes/AccountHelper.cls

```diff
@@ -28,19 +28,54 @@ Always ensure that your helper methods are bulk-safe to handle multiple records
 */
 public with sharing class AccountHelper {
     
-    public void setTypeProspect(SObject objects){
+    public static void updateAccountFileds(List<Account> newAccounts) {
+        List<Contact> contacts = new List<Contact>();
 
-    }
+        for (Account acc : newAccounts){
+
+            //Set the account type to prospect.
+            //When an account is inserted change the account type to 'Prospect' if there is no value in the type field.
+            if(String.isBlank(acc.Type)) {
+                acc.Type = 'Prospect';
+            }            
+            
+
+            //Set the account rating to hot.
+            //When an account is inserted set the rating to 'Hot' if the Phone, Website, and Fax ALL have a value.
+            if(String.isNotBlank(acc.Phone) && String.isNotBlank(acc.Fax) && String.isNotBlank(acc.Fax)) {
+                acc.Rating = 'Hot';
+            }
+            
 
-    void addressCopy(){
```

Comment:
The thing you are doing here. is setting a default contact. I would name that method what it does and call it from the Trigger Handler,  calling helper.afterAccount from the afterInsert method is confusing. 

```
 public override void handleAfterInsert(){
        createDefaultContact();
    }
    ```